code/acting/src/acting/DummyTrajectoryPublisher.py

```diff
@@ -96,6 +95,7 @@ def run(self):
 
         def loop(timer_event=None):
             # Continuously update path
+            self.updated_trajectory(self.initial_trajectory)
             self.trajectory_publisher.publish(self.path_msg)
```

Comment:
It is not necessary to continually publish the path. I did so, because I would expect that the path would change from time to time as new information becomes available.